src/commands/oas2json.js

```diff
@@ -1,40 +1,59 @@
 import { Command } from 'commander'
+import filenamify from 'filenamify'
 import fs from 'fs-extra'
-import YAML from 'yaml'
+import _get from 'lodash.get'
+import _trimStart from 'lodash.trimstart'
 import path from 'path'
-import { exit } from 'process'
 import pino from 'pino'
+import { exit } from 'process'
+import YAML from 'yaml'
+
 import { fromSchema } from '../utils/openapi-schema-to-json-schema-wrapper.cjs'
 
 const COMPONENT_REF_REGEXP = /#\/components\/schemas\/[^"]+/g
 
-export const adaptSchema = (generatedSchema, name) => {
+export const adaptSchema = (generatedSchema, name, filename) => {
   delete generatedSchema.$schema
   generatedSchema.title = name
-  generatedSchema.$id = `${name}.json`
+  generatedSchema.$id = `${filename}.json`
 
   if (generatedSchema.format?.includes('date')) {
     generatedSchema.tsType = 'Date'
   }
 }
 
-const processSchema = (name, schema, schemasPath) => {
-  const generatedSchema = fromSchema(schema)
-  adaptSchema(generatedSchema, name)
+const processSchema = (name, schema, schemasPath, logger, dirname = '') => {
+  let generatedSchema
+  try {
+    generatedSchema = fromSchema(schema)
```

Comment:
I recall that `fromSchema` doesn't convert definitions by default:
https://github.com/openapi-contrib/openapi-schema-to-json-schema#definitionkeywords-array

I believe we should provide the paths to be converted as `definitionKeywords`.

BTW, if true, we should review the OpenAPI used in the tests.